docs/token_transferring/neonpass_usage.md

```diff
@@ -57,43 +49,48 @@ The following is an example of transferring NEON tokens from Neon EVM to Solana
 
 > For further help, see our [wallet setup walkthrough](/docs/wallet/metamask_setup).
 
-### Step 2: Connect wallets to NeonPass 
+### Step 2: Connect wallets to NeonPass
 
 :::tip
 
 Remember to use a browser to which both your Solana- and EVM-compatible wallets are attached.
 
 :::
 
-<img src={image1} width="450" style={{ display: 'block', margin: '10px auto' }} />
+<img src={image3} width="450" style={{ display: 'block', margin: '10px auto' }} />
+
+2.1 Select the network you want to connect to from the dropdown in the top right corner.
 
+<img src={image1} width="450" style={{ display: 'block', margin: '10px auto' }} />
 
-2.1 Click **Connect Wallet** to connect your Solana-compatible wallet to NeonPass. Follow the login procedure in your wallet's popup window and ensure it's connected to the network you require (e.g. Mainnet). 
+2.2 Click **Connect Wallet** to connect your Solana-compatible wallet to NeonPass. Follow the login procedure in your wallet's popup window and ensure it's connected to the network you require (e.g. Mainnet).
 
 :::tip
 
 Ensure you have enough SOL/NEON to pay the withdrawal approval fee.
 
 :::
 
-2.2 Repeat this step for your EVM-compatible wallet. 
+2.3 Repeat this step for your EVM-compatible wallet.
 
 After successfully connecting your wallets to NeonPass, the **Connect Wallet** text changes to the name of the network (Solana or Neon), and the public key of your accounts is displayed.
 
 ### Step 3: Conduct transfer
 
-3.1 Use the arrow to determine the direction of transfer (from Solana to Neon EVM or vice versa).
+<Tabs>
+  <TabItem value="neontosolana" label="Neon EVM to Solana" default>
+
+3.1 Use the arrow to determine the direction of transfer (from Neon EVM to Solana).
 
 <img src={image2} width="450" style={{ display: 'block', margin: '10px auto' }} />
 
 3.2 Use the drop-down to select the token you want to transfer and enter the amount you want to send.
 
-3.3 Click **Transfer**. 
+3.3 Click **Transfer**.
 
 3.4 Confirm the transaction in your sending wallet.
```

Comment:
@sukanyaparashar as per the comment that existed in the metadata of the page >> I believe this flow. As in, confirmation is only required in the sending wallet. However, your two 3.4 steps differ >> the "Solana to Neon EVM" tab (line 128) says: "The transaction must then be confirmed in both the MetaMask and Phantom wallet windows which will pop up." Please confirm that these differences are valid and intentional.

Reply:
This should be changed in both the contexts actually. We only confirm the transaction in one wallet from where we are transferring the tokens. I will change this in both the contexts.

Reply:
So, if you look at the metadata of the page, you will see if a previous author/reviewer has spotted potential issues in the comments (I deleted the comment on my previous commit in favor of moving it to the review here) >> Please always read those comments to understand the challenges of a page. The comment had identified this exact issue.

Reply:
Yeah! I have missed reading the comment. I will keep that in mind from next time :)

Reply:
I have also fixed the todo as per the page's metadata (relocating images to Docusaurus standard)